### PR TITLE
feat: publish the first version of the package to npm

### PR DESCRIPTION
**Pull request by mohandast52**
* publish the first version to the NPM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "autonolas-frontend-library",
```
[the remark this answers is not in the document]
Comment (dvilelaf) [reply package.json:2]: I think that one is for @DavidMinarsch but I'd say @valory-xyz/autonolas-frontend-library for coherence.

File: README.md [hunk 2]
```diff
@@ -37,11 +36,3 @@ Once you are done developing in `autonolas-frontend-library`, you can follow the
...
 2. In `autonolas-protocol-frontend`, run `yarn install autonolas-frontend-library` to fetch the latest published version from GitHub.
 3. In `autonolas-frontend-library`, run `yarn unlink` to delete the symlink created by yarn. You may require sudo permissions to run this command.
-
-## Steps to update autonolas-frontend-library package to latest version
-
-1. Find the entry for `autonolas-frontend-library` in `package.json`
-2. Copy the short hash of the latest commit on [autonolas-frontend-library master](https://github.com/valory-xyz/autonolas-frontend-library/commits/main)
-3. Paste the hash into the github link that controls the `autonolas-frontend-library` version in `package.json`
-4. Run `yarn` to also update package-lock.json
-5. Open a PR with these changes.
```
Comment (dvilelaf) [inline README.md:47]: Why removed? Is it not neccesary anymore?

**Review by dvilelaf — COMMENTED** [review]
LGTM, ping David on the naming issue before publishing

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply package.json:2]: `@autonolas/frontend-library`

**Review by dvilelaf — APPROVED** [review]

**Pull request merged**